R/stat_stepribbon.R

```diff
@@ -0,0 +1,77 @@
+#' Step ribbon statistic
+#'
+#' Provides stairstep values for ribbon plots
+#'
+#' @name stat_stepribbon
+#' @inheritParams ggplot2::geom_ribbon
+#' @param geom which geom to use; defaults to "`ribbon`"
+#' @param direction `hv` for horizontal-vertical steps, `vh` for
+#'   vertical-horizontal steps
+#' @references [https://groups.google.com/forum/?fromgroups=#!topic/ggplot2/9cFWHaH1CPs]()
```

Comment:
do we need the ( ) at the end?

Reply:
it's a standard markdown syntax for a link. you can check the rendered help file to see how it appears